pipelines/ApPipe.yaml

```diff
@@ -49,31 +51,32 @@ subsets:
       - characterizeImage
       - calibrate
       - retrieveTemplate
-      - imageDifference
+      - subtractImages
+      - detectAndMeasure
       - transformDiaSrcCat
       - diaPipe
     description: >
       An alias of ApPipe to use in higher-level pipelines.
 contracts:
   # DiaPipelineTask needs diaSource fluxes, catalogs, warped exposures, and difference exposures
-  - imageDifference.doMeasurement is True
-  - imageDifference.doWriteSources is True
-  - imageDifference.doWriteWarpedExp is True
-  - imageDifference.doWriteSubtractedExp is True
```

Comment:
If you are removing these flags, please remove the comment that describes them as well.